sktime/classification/feature_based/_signature_classifier.py

```diff
@@ -102,7 +102,7 @@ class SignatureClassifier(BaseClassifier):
     """
 
     _tags = {
-        "coerce-X-to-numpy": False,
+        "convert_X_to_numpy": False,
```

Comment:
this is not a tag that exists, is it?

Reply:
good spot, thought I had caught those, wonder if tests will catch it?